### ISSUE TITLE
The plugin encountered an error, `(*GRPCProvider).ConfigureProvider` call.

### ISSUE THREAD

**Issue by TechDufus** [Telmate/terraform-provider-proxmox#762]
After updating from provider version `2.9.11` to `2.9.14` I get the following error when trying to `terraform plan`. Downgrading to `2.9.11` takes removes this error:
```error

Planning failed. Terraform encountered an error while generating this plan.

│ Error: Plugin did not respond
│ 
│   with provider["registry.terraform.io/telmate/proxmox"],
│   on provider.tf line 26, in provider "proxmox":
│   26: provider "proxmox" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ConfigureProvider call.
│ The plugin logs may contain more details.

```

There is no change to my provider config, and all modules require the same version. Here is how my provider is configured:
```hcl
provider "proxmox" {
  pm_api_url      = var.proxmox_api_url
  pm_user         = var.pm_user
  pm_password     = var.pm_password
  pm_tls_insecure = true
}
```

Thoughts? Do I have the dumb? Or is there something going on here that I don't see?

**Comment by TechDufus** [comment]
> Hello, Which version of terraform are you using ? Are you, by any chance, on a M1 chips Mac ?

I'm using `v1.4.6`. 

No, I'm using an AMD 5950x.

**Comment by hwts** [comment]
I am having the same problem. I had to roll back to version 2.9.11

**Comment by ammmze** [comment]
Finally taking some time to debug this. I've had a PR open for my home lab failing with this for 4 months now 😂. So with a combination of setting env var `export TF_LOG=TRACE` and `pm_debug = true` in the provider config I found that it looked like I was getting permission errors. It seems the permissions needed have changed. I saw mention of `Sys.Audit` in the logs. Anyways, so I just updated the role that I created originally when setting up the provider with the following:

```shell
pveum role modify TerraformProv -privs "Datastore.AllocateSpace Datastore.Audit Pool.Allocate Sys.Audit Sys.Console Sys.Modify VM.Allocate VM.Audit VM.Clone VM.Config.CDROM VM.Config.Cloudinit VM.Config.CPU VM.Config.Disk VM.Config.HWType VM.Config.Memory VM.Config.Network VM.Config.Options VM.Migrate VM.Monitor VM.PowerMgmt"
```

Doing that got me past the GRPCProvider error and now have a different error around some missing bool in the disk config, which appears to be issue #702
